backend/src/libs/packages/database/schema/tables-schema.ts

```diff
@@ -60,4 +63,45 @@ const business = pgTable('business_details', {
   updatedAt: timestamp('updated_at').notNull().defaultNow(),
 });
 
-export { business, groups, users, usersRelations };
+const businessRelations = relations(users, ({ many }) => ({
+  orders: many(orders),
+}));
+
+const orderStatus = pgEnum('status', ORDER_STATUSES);
```

Comment:
```suggestion
const orderStatus = pgEnum('order_status', ORDER_STATUSES);
```
to avoid collisions in the future - there might be many different statuses

Reply:
fixed